settings/common.py

```diff
@@ -11,10 +11,12 @@
 """
 
 import os
-
+import sys
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+APPS_DIR = os.path.join(BASE_DIR, 'apps')
 
+sys.path.append(APPS_DIR)
```

Comment:
I don't think we would need this since `apps` directory is already there on PYTHONPATH(root of the project).